accounts.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	// "errors"
+	// "time"
+
+	"errors"
+
+	"github.com/distributeddesigns/currency"
+	// "github.com/op/go-logging"
+)
+
+// Account : State of a particular Account
+type Account struct {
+	Balance currency.Currency
```

Comment:
TIL that `go build` will enforce pass by reference if an object has a mutex. Thankfully, that's how we handle `account`s but it could have been a non-trivial refactor.